ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_attribute.py

```diff
@@ -86,6 +86,31 @@ def test_paddle_is_floating_point(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.tensor.attribute.rank",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+)
+def test_paddle_rank(
+    *,
+    dtype_and_x,
+    on_device,
+    fn_tree,
+    frontend,
```

Comment:
Can you add another argument `backend_fw` here. It was recently added and It will be generated by `handle_frontend_test` so no need to worry about its generation.